[doc] Adding release version for Core Web Vitals (#10361)
DataDog/documentation: content/en/real_user_monitoring/browser/monitoring_page_performance.md

- **RUM waterfall**: accessible for every single RUM view event in the [RUM Explorer][3], it lets you troubleshoot the performance of a specific page view. It shows how your website assets and resources, long tasks, and frontend errors affect performance for your end users, at the page level.

### Core Web Vitals
<div class="alert alert-warning"> 

>   <strong>Note:</strong> The Core Web Vitals metrics have been introduced in Datadog since [`@datadog/browser-rum`](https://github.com/DataDog/browser-sdk) package version 2.0.0

Reviewer: Because everything is wrapped in a div element here, all styling has to be done in html, the markdown won't be processed.
```suggestion
  <strong>Note:</strong> The Core Web Vitals metrics are available in Datadog from <a href="https://github.com/DataDog/browser-sdk">@datadog/browser-rum</a> package version 2.0.0 and newer.
```
